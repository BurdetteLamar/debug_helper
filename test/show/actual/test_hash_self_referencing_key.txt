---
Hash (message='test_hash_self_referencing_key'):
  Hash#size: 1
  Hash#default: 
  Hash#default_proc: 
  Pair 0:
    Key: Hash {{...}=>0}
    Value: Integer 0
