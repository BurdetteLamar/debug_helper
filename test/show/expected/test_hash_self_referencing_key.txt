---
Hash (message='test_hash_self_referencing_key'):
  size: 1
  default: 
  default_proc: 
  Pair 0:
    Key: Hash {{...}=>0}
    Value: Fixnum 0
